### PR TITLE
LPS-112110 Make Storage Adapter value localized

### PR DESCRIPTION
**Pull request by carolmariaabb**

### CODE REVIEW

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by natocesarrego** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-instance/src/main/java/com/liferay/dynamic/data/mapping/data/provider/instance/DDMStorageTypesDataProvider.java [hunk 1]
```diff
@@ -41,13 +45,34 @@
...
+
+			String defaultValue = LanguageUtil.get(
+				httpServletRequest, storageType);
+
+			keyValuePairs.add(
+				new KeyValuePair(
+					storageType,
+					LanguageUtil.get(
+						httpServletRequest, storageType + "[stands-for]",
+						defaultValue)));
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-instance/src/main/java/com/liferay/dynamic/data/mapping/data/provider/instance/DDMStorageTypesDataProvider.java:75]: Great job, @carolmariaabb! 😄 👏

**Comment by natocesarrego** [comment]
Pull request submitted to https://github.com/natocesarrego/liferay-portal/pull/496. [See changes here.](https://github.com/natocesarrego/liferay-portal/compare/3a570e262373~3...3a570e262373) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**